dashboard/app/models/level.rb

```diff
@@ -189,9 +189,9 @@ def delete_custom_level_file
   TYPES_WITHOUT_IDEAL_LEVEL_SOURCE =
     ['Unplugged', # no solutions
      'TextMatch', 'Multi', 'External', 'Match', 'ContractMatch', # dsl defined, covered in dsl
-     'Applab', # all applab are freeplay
+     'Applab', 'Gamelab', # all applab and gamelab are freeplay
```

Comment:
Don't fix yet, but at some point we should formally decide whether gamelab is one word or two (in code), and capitalize accordingly.